content/th/intro.md

```diff
@@ -1,12 +1,12 @@
 บทนำ
 ============
 
-ในยุคสมัยใหม่ ซอฟต์แวร์ถูกส่งมอบทั่วไปเป็นบริการ: เรียกว่า *web apps*, หรือ *software-as-service*. twelve-factor app เป็นหลัการสำหรับสร้างแอพพลิเคชัน software-as-a-service ที่:
+ในยุคสมัยใหม่ ซอฟต์แวร์ถูกส่งมอบทั่วไปเป็นบริการ: เรียกว่า *web apps*, หรือ *software-as-service*. twelve-factor app เป็นหลักการสำหรับสร้างแอพพลิเคชัน software-as-a-service ที่:
```

Comment:
QQ: in Thai, does it make sense to call 12 Factor a "principle" instead of a "framework"? Trying to understand the change.

Reply:
หลักการ = principle
For framework, we don't have direct word. Usually we just use transliteration = เฟรมเวิร์ก